display/pfobjects.py

```diff
@@ -34,11 +29,23 @@ def __init__(self, cluster):
         inners = [self.inner_xy, self.inner_yz, self.inner_xz,
                   self.inner_thetaphi]
         for contour in contours:
-            contour.SetLineColor(color)
-            contour.SetFillStyle(0)
-        for inner in inners: 
-            inner.SetFillColor(color)
-            inner.SetFillStyle(3002)
+           if cluster.particle:
```

Comment:
why moving the color definition inside the loop? I understand that you would like slightly different colors for photons and electons, but couldn't you just add them to the existing code? 
in other words, please revive the code removed between lines 14-18 and modify it to suit your needs